HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -2939,6 +2946,7 @@ def self.create_dx_heating_coil(model, obj_name, heating_system, max_rated_fan_c
         end
         htg_coil.setRatedTotalHeatingCapacity(UnitConversions.convert(htg_ap.heat_rated_capacities_gross[i], 'Btu/hr', 'W'))
         htg_coil.setRatedAirFlowRate(calc_rated_airflow(htg_ap.heat_rated_capacities_net[i], htg_ap.heat_rated_cfm_per_ton[0]))
+        defrost_time_fraction = 0.1 if heating_system.advanced_defrost_approach # 6min/hr
```

Comment:
@yzhou601, Would it be better to test for the number of compressor speeds directly? Sorry if I'm not understanding the code correctly, but will detailed_performance_data always be filled for variable speed?

Reply:
Sorry Jon I didn't quite understand the question here. These `defrost_time_fraction` is set under [num_speed conditions](https://github.com/NREL/OpenStudio-HPXML/blob/a4c855ceef39b5313db22db6bac5af60397ee263/HPXMLtoOpenStudio/resources/hvac.rb#L2940-L2949).

Reply:
Oh, I see. Sorry I missed that. This looks good to me.